docs/source/guide/pec-1-intro.myst

```diff
@@ -0,0 +1,142 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# How do I use PEC?
+
+Probabilistic Error Cancellation (PEC) is an error mitigation technique for improving the estimation
+of expectation values (see [What is the theory behind PEC?](pec-5-theory.myst)). To be applied, PEC requires the 
+full knowledge of the noise model. In particular, the user must know how to represent ideal gates as linear 
+combinations of noisy implementable gates. This knowledge is exploited by Mitiq
+to probabilistically remove the bias error from noisy expectation values.
+
++++
+
+The following code is compatible with the multiple frontends. The next code cell can be used for selecting one of them.
+
+```{code-cell} ipython3
+frontend = "qiskit"  # Supported: "cirq", "qiskit", "pyquil", "braket", "pennylane".
+```
+
+## Define the circuit of interest
+
++++
+
+We first define the circuit of interest. For simplicity, in this example we use
+a single-qubit randomized-benchmarking circuit whose ideal execution is equivalent to the
+identity operation.
+
+```{code-cell} ipython3
+from mitiq import benchmarks
+
+circuit = benchmarks.generate_rb_circuits(
+  n_qubits=1, num_cliffords=2, return_type = frontend,
+)[0]
+
+print(circuit)
+```
+In the code cell above, we used the function {func}`~mitiq.benchmarks.randomized_benchmarking.generate_rb_circuits()`, in which `num_cliffords` is the 
+number of random Clifford group elements and is proportional to the depth of the circuit.
+
+## Define a noisy executor
```

Comment:
Remove "noisy". We should always assume people are using Mitiq to run on hardware which is inherently noisy.